src/Spice86.Core/Emulator/OperatingSystem/Dos.cs

```diff
@@ -226,6 +239,30 @@ private VirtualFileBase[] AddDefaultDevices(State state, KeyboardInt16Handler ke
         return [nulDevice, consoleDevice, printerDevice];
     }
 
+    /// <summary>
+    /// Allocates a small code area in the DOS device segment for device-private ASM stubs. <br/>
```

Comment:
Cant this reside with the other interrupts code? This way no need to allocate anything